archive/imo/imo2008_q2.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright (c) 2021 Manuel Candales. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Manuel Candales
+-/
+import data.real.basic
+
+/-!
+# IMO 2008 Q2
```

Comment:
State in the module doc that you are only doing part a). Possibly also rename the file to `_q2a`.

Reply:
Added part b in new commit